webapp/channels/src/packages/mattermost-redux/src/selectors/entities/scheduled_posts.ts

```diff
@@ -16,8 +17,8 @@ export function makeGetScheduledPostsByTeam(): (state: GlobalState, teamId: stri
         'makeGetScheduledPostsByTeam',
         (state: GlobalState) => state,
```

Comment:
Out of the changes scope, but still.. can you verify this?

My guess is that state changes with any change on state, so any unrelated change on the state will make this return a different value, and therefore, create a new `scheduledPostList`in this selector, making the selector less useful as it should.

Looking at the implementation, it is only used to get `state.entities.scheduledPosts.byId`, so we can change it to get just that on here.

Reply:
Done.